src/core/dune

```
@@ -22,10 +22,10 @@
   liquidsoap_lang
   menhirLib
   (select
-    charset.ml
+    charset_impl.ml
     from
-    (camomile -> charset.camomile.ml)
-    (-> charset.native.ml)
+    (camomile -> charset_impl.camomile.ml)
+    (rosetta -> charset_impl.rosetta.ml)
```

Why not keep the "native" implementation? I agree that we should ask for rosetta or camomil in the opam file, but we could still keep the native implementation around to still be able to compile with dune if we have a problem with rosetta, or want to have a small binary etc?
```suggestion
    (camomile -> charset_impl.camomile.ml)
    (rosetta -> charset_impl.rosetta.ml)
    (-> charset_impl.native.ml)
```